packages/EasyApiPlatform/tests/Filter/AbstractFilterTestCase.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyApiPlatform\Tests\Filter;
+
+use ApiPlatform\Doctrine\Orm\Filter\FilterInterface;
+use ApiPlatform\Doctrine\Orm\Util\QueryNameGenerator;
+use Doctrine\Bundle\DoctrineBundle\Registry;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+use EonX\EasyApiPlatform\Tests\Stubs\App\ApiResource\Dummy;
+use EonX\EasyApiPlatform\Tests\Stubs\App\Kernel\TestKernel;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+abstract class AbstractFilterTestCase extends KernelTestCase
```

Comment:
@alexndlm @itorgov Let's continue the discussion here.
At the first glance, the current coverage looks too complicated. If I understand correctly, there is no coverage at all for the `VirtualSearchFilter`?

Reply:
> > Let's mark `\EonX\EasyCore\Bridge\Symfony\ApiPlatform\Filter\VirtualSearchFilter` as deprecated.
> 
> @akl-95, I believe we should not do this.
> 
> `VirtualSearchFilter` works with 2.6 This filter works for 2.7

It will not work with `2.6` due to different namespaces.

Reply:
Is the question still actual? If yes, would you please write a new comment? I'm afraid I've lost the point.

Reply:
> If I understand correctly, there is no coverage at all for the VirtualSearchFilter?

Reply:
@akl-95 @alexndlm And I'm still don't want to have all these new test files copied from API Platform. Personally I don't like the idea to maintain them.

Reply:
These test are really easy maintainable.

Reply:
Not really. We will need to remember about this "fake" app. Update its dependencies time to time, deprecated config params, etc.
If we really need/want to have such a fake app to test our custom filters I suggest to have it for the whole EasyMonorepo. I really don't like to have a fake app for each package.

Reply:
We already have many test apps. They differ only in the way they are configured.

We need more huge configurations for the API Platform - that's all differences.

<img width="1440" alt="Снимок экрана 2023-02-27 в 08 29 21" src="https://user-images.githubusercontent.com/6824784/221501156-4778bfc6-6cb9-4b33-bb1b-42f706a31371.png">

Reply:
> We already have many test apps.

Huh, usually it is just one file. I wouldn't name call it "app" :-)